charts/aws-efs-csi-driver/CHANGELOG.md

```diff
@@ -0,0 +1,69 @@
+# Helm chart
+
+# v2.0.0
+
+## Breaking changes
```

Comment:
You don't mention moving nodeSelector and resources under node.  It is probably also worth noting that you fixed a bug where the controller was using podAnnotations and tolerations from the node section and is now using the correct values from the controller section.  It might be worth putting a section in the readme under the helm instructions that point here for upgrading from versions prior to 2.0.0

Reply:
Tried my best to clarify everything, tell me if it's not enough